api-reference/10 UI Components/dxDateRangeBox/1 Configuration/endDateText.md

```diff
@@ -4,7 +4,7 @@ type: String
 ---
 ---
 ##### shortDescription
-The read-only property that holds the text displayed by the end date input field.
+The read-only property that holds the text displayed by the end date input field. 
```

Comment:
Can we just say "returns" instead of "read-only property that holds"? That is our usual word for such cases and it implies that the property is "read-only". 

```suggestion
Returns the text displayed by the end date input field. 
```